src/router/router.ts

```diff
@@ -69,7 +69,18 @@ export class Router {
    * @returns
    */
   private getLocalizedRoutes(locale: string) {
-    return this.schema.routes[locale] || []
+    return (
+      this.schema.routes[locale]?.sort((a, b) => {
```

Comment:
@svobik7 Should I create new test suite, or expand the existing one for router?

Reply:
I am wondering, because I have tried adding additional routes to schema mock, and tests stopped passing, despite not adding any new cases to the suite

Reply:
Also, the `getLocalizedRoutes` is private method, so I think in order to write test, I would need to expose it to the outer world, which is not good.

Reply:
That is true? So how about extending current router test for getHref? https://github.com/svobik7/next-roots/blob/master/src/router/router.test.ts

Basically my idea is to simulate wrong order by passing adjusted schema and test that I am getting right href.

Reply:
That makes sense! Will add them today

Reply:
@svobik7 Done 😄

Reply:
Hi @pzpsofficial thank you for your quick reply. I am sorry I misled you a bit. The test cases should have been added to `getRouteByHref` https://github.com/svobik7/next-roots/blob/master/src/router/router.test.ts#L94.

But no worries I will add them before publishing. Thank you for your contribution. I am gonna publish the new version in an hour.